custom/ilsgateway/balance.py

```diff
@@ -67,7 +69,11 @@ def validate_sms_users(self):
                 elif vn and vn.domain != self.domain:
                     description += "Phone number already assigned on domain {}, ".format(vn.domain)
                 elif not vn or not vn.verified:
-                    description += "Phone number not verified, "
+                    try:
```

Comment:
This change allows to verify phone number that was deleted from some inactive user.